### PR TITLE
Formating CHM API + remove obsolete properties

### PR DESCRIPTION
**Pull request by patrikmiksovsky95**

### CODE REVIEW

**Comment by patrikmiksovsky95** [comment]
@jirrick there are just tabulators you can hide them as did Ondra by hiding spaces

File: mews-api.md [hunk 1]
```diff
@@ -499,28 +499,28 @@ This is example of a _successful_ response. In case an error occurred, the respo
 
 ### Request ARI update
 
-\[`async`\] This method allows channel manager to request an ARI data update for certain space types and rate plans in addition to the changes automatically sent in the next Delta update. The requested data will be sent by Mews asynchronously via push operations of channel manager API.
+\[`async`\] This method allows channel manager to request an ARI data update for certain space types and rate plans in addition to the changes automatically sent in the next [Delta](channel-manager-api.md#delta-inventory-update-mode) update. The requested data will be sent by Mews asynchronously via push operations of channel manager API.
```
Comment (allisonchihak) [inline mews-api.md]:
```suggestion
\[`async`\] This method allows channel manager to request an ARI data update for certain space types and rate plans in addition to the changes automatically sent in the next [delta](channel-manager-api.md#delta-inventory-update-mode) update. The requested data will be sent by Mews asynchronously via push operations of channel manager API in the next delta update.
```

File: mews-api.md [hunk 2]
```diff
@@ -62,14 +62,14 @@ The property is configured to accept following test credit cards:
 
 ### Get Properties
 
-\[`sync`\] This method is used to obtain properties based on the `email` of an employee of the Mews enterprise (hotel, hostel or apartment group). It is required that the `email` is verified to belong to an employee on whose behalf the API call is made. Response returns list of all enterprises, where employee has access in Mews.
+\[`sync`\] This method is used to obtain properties based on the `email` of an employee of the Mews enterprise (hotel, hostel or apartment group). It is required that the `email` is verified to belong to an employee on whose behalf the API call is made. Response returns list of all enterprises, where employee has access in Mews, and the `connectionToken`s of all connections for the requesting `clientToken`.
```
[the remark this answers is not in the document]
Comment (patrikmiksovsky95) [reply mews-api.md]: I think even sense what is there is clear for me

File: mews-api.md [hunk 3]
```diff
@@ -563,203 +563,189 @@ There are certain rules that need to be followed in order for Mews to process th
   * The whole group is uniquely identified by `channelManagerId` in Mews and in the channel manager extranet.
   * Each reservation should have a unique code within the group. The same code for the reservation should be provided in any following modification message.
   * Each reservation in the group can have different `start`, `end`, `spaceTypeCode`, `ratePlanCode`.
-* Group total cost `group`.`totalAmount` is the sum of all `reservation`.`totalAmount` objects in the group, which is the sum of all night amounts and total amounts of all `extras` of the `reservation`.
-  * If for any reason the sum of the `reservation`.`totalAmount` objects is different, Mews will automatically distribute the missing/additional amount to the nights, so the `group`.`totalAmount` is achieved.
+* Group total cost `group.totalAmount` is the sum of all `reservation.totalAmount` objects in the group, which is the sum of all night amounts and total amounts of all `extras` of the `reservation.`
```
Comment (allisonchihak) [inline mews-api.md]:
```suggestion
* Group total cost `group.totalAmount` is the sum of all `reservation.totalAmount` objects in the group, which is the sum of all night amounts and total amounts of all `extras` of the `reservation`.
```

**Pull request merged**